src/Avalonia.Base/Threading/DispatcherOptions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace Avalonia.Threading;
```

Comment:
We usually keep other **Options classes in `Avalonia` namespace.
SkiaOptions as an example: https://github.com/AvaloniaUI/Avalonia/blob/master/src/Skia/Avalonia.Skia/SkiaOptions.cs#L4

So it won't require any extra usings for app builder.

Reply:
We don't really seem to be consistent in this. A couple of exceptions I've seen are:

- `Avalonia.Rendering.Composition.CompositionOptions`
-  `Avalonia.LinuxFramebuffer.LinuxFramebufferPlatformOptions`

However, yes I can move this class to the root namespace.